.github/workflows/test.yml

```
@@ -36,6 +36,17 @@ jobs:
         with:
           fail_ci_if_error: true
 
+  test-docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build docker image
+        run: docker build -t draco .
+
+      - name: Run `make all`
+        run: docker run --rm draco bash -c "make all"
```

remind me, do we need to say "all"?

```suggestion
        run: docker run --rm draco bash -c "make"
```

---

No, we don't, thanks for the hint!